src/olympia/search/filters.py

```diff
@@ -141,12 +141,25 @@ class FooAddonThresholdQueryParam(AddonThresholdQueryParam):
         '__gte': 'gte',
         '': 'eq',
     }
+    valid_types = (int, float, str)
+
+    def convert_value(self, value):
+        if value.isdecimal():
+            converter = int
+        elif value.replace('.', '', 1).isdecimal():
```

Comment:
Technically this is a backwards incompatible change - previously any [valid float string](https://docs.python.org/3/library/functions.html#float) would be accepted - but 🤷.